app/assets/stylesheets/application.scss

```diff
@@ -15,7 +15,12 @@
 @import "@ministryofjustice/frontend/moj/utilities/hidden";
 @import "@ministryofjustice/frontend/moj/components/multi-file-upload/multi-file-upload";
 @import "@ministryofjustice/frontend/moj/components/banner/banner";
+@import "@ministryofjustice/frontend/moj/objects/_scrollable-pane";
 
 .govuk-button-group.confirmation-button-group > form {
     display: inline-flex;
 }
+
+.fixed-to-right {
```

Comment:
So this is a weird one that seems to only affect Chrome and I wonder why this isn't upstreamed. `govuk-visually-hidden` should probably also have `inset: 0` applied to it, I might query that but in the meantime yeah this works.

Worth adding a comment maybe?

Reply:
I was wondering that too, but in the `govuk-visually-hidden` we have `position: absolute` and no `inset` set. It only acts weird when placed in a `scrollable-pane` which is MOJ component and not GOVUK component. May be thats why it's never been flagged.

Reply:
`inset` is just shorthand for `top`, `left`, `bottom` and `right`, so just a more complete version of what you had.

And yes perhaps, I'll email them shortly.